test/e2e/test-amp-bind-video.js

```diff
@@ -15,9 +15,10 @@
  */
 
 describes.endtoend(
-  'amp-bind',
+  'amp-bind with <amp-video>',
   {
     testUrl: 'http://localhost:8000/test/fixtures/e2e/amp-bind/bind-video.html',
+    environments: 'ampdoc-amp4ads-preset',
```

Comment:
This env isn't defined? Also should be an array i.e. `['a4a-fie']`

Reply:
I introduced a concept of preset, which represent a collection of envs. Let me know if it works for you

Reply:
oh I see, looks good